### PR TITLE
Using Environment Variable SENSORSAFRICA_DATABASE_URL 

### PR DESCRIPTION
**Pull request by karimkawambwa**
## Description

Switching to using one variable db url.

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: pylama.ini [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[pylama]
+ignore = E501
```
Comment (kilemensi) [inline pylama.ini]: New line

File: sensorsafrica/settings.py [hunk 2]
```diff
@@ -21,112 +22,108 @@
...
-        'ENGINE': 'django.db.backends.postgresql',
-        'NAME': os.environ.get('SENSORSAFRICA_DBNAME', 'sensorsafrica'),
-        'USER': os.environ.get('SENSORSAFRICA_DBUSER', 'Karim'),
-        'PASSWORD': os.environ.get('SENSORSAFRICA_DBPASS', ''),
-        'HOST': os.environ.get('SENSORSAFRICA_DBHOST', 'localhost'),
-        'PORT': os.environ.get('SENSORSAFRICA_DBPORT', '5432'),
-    }
-}
+DATABASE_URL = os.getenv(
+    "SENSORSAFRICA_DATABASE_URL", "postgres://Karim:@localhost:5432/sensorsafrica"
```
Comment (kilemensi) [inline sensorsafrica/settings.py]: 1. `Karim` is a bad default username.
 2. Empty password is also bad default.

Why not `sensorsafrica:sensorsafrica`?

File: sensorsafrica/settings.py [hunk 3]
```diff
@@ -21,112 +22,108 @@
 # See https://docs.djangoproject.com/en/1.10/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = '-kc8keig#xrdhi1l$rrj&s*s@3pz*4he)8u8h^w$2-_4y6@z3g'
+SECRET_KEY = os.getenv(
+    "SENSOSAFRICA_SECRET_KEY", "-kc8keig#xrdhi1l$rrj&s*s@3pz*4he)8u8h^w$2-_4y6@z3g"
```
Comment (kilemensi) [inline sensorsafrica/settings.py]: `SENSORSAFRICA_SECRET_KEY`

File: sensorsafrica/settings.py [hunk 4]
```diff
@@ -21,112 +22,108 @@
...
+)
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = os.getenv("SENSOSAFRICA_DEBUG", "True") == "True"
```
Comment (kilemensi) [inline sensorsafrica/settings.py]: `SENSORSAFRICA_DEBUG`

**Comment by karimkawambwa** [comment]
@kilemensi 👀

**Pull request merged**